azure/README.md

```diff
@@ -47,7 +47,7 @@ For security reasons, initial configuration from another machine is disabled, bo
 
 ### IV. Install the Guardian Connector software stack
 
-Install the app stack by following [`caprover/README.md`](https://github.com/ConservationMetrics/gc-forge/blob/main/caprover/README.md).
+Install the app stack by following [`caprover/INSTALL_GC_STACK.md`](caprover/INSTALL_GC_STACK.md).
```

Comment:
```suggestion
Install the app stack by following [`/caprover/INSTALL_GC_STACK.md`](/caprover/INSTALL_GC_STACK.md).
```

Reply:
Woah, do absolute paths work?

Even if they do work correctly in GitHub, should I be wary about ...?
- local browsing (from git source)
- GitHub Pages

Your comment is correct to point out that I had the links wrong in either case.

Reply:
They seem to! Also a pattern that works in Docusaurus.

But, honestly, just go for `../` since that's a convention I see is already in use across the repo.